### PR TITLE
User can now click history and segue to history detail

### PR DESCRIPTION
**Pull request by devin1727**

### CODE REVIEW

File: monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift [hunk 1]
```diff
@@ -31,6 +31,7 @@ struct TodoListTabItem: View {
 						isExpanded: self.selection.contains(taskStatuses[index]),
 						icon: statusIcon[index]
 					)
+                    .animation(Animation.easeIn)
```
Comment (chrsomle) [inline monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift:34]: ini bisa .animation(.easeIn) aja

File: monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift [hunk 2]
```diff
@@ -60,7 +61,7 @@ struct TodoListTabItem: View {
 
 	func getFilteredTask(filter: TaskStatus) -> [Task] {
 		let result = taskViewModel.tasks.filter { $0.status == filter}
```
Comment (chrsomle) [inline monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift:63]: swiftlint eyy { $0.status == filter }

**Review by chrsomle — CHANGES_REQUESTED** [review]
Ada conflict sama file HistoryView.swift di branch dev

File: monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift (hunk 1 above)
Comment (devin1727) [reply monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift:34]: hoo ini coba coba, animasinya g mau muncul, g tau kenapa

File: monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift (hunk 2 above)
Comment (devin1727) [reply monitool/Views/iPhone/Components/Tab Items/TodoListTabItem.swift:63]: Ha? maskdunya?

**Review by chrsomle — APPROVED** [review]

**Pull request merged**